src/main/resources/types.graphqls

```diff
@@ -67,15 +67,15 @@ type Annotations {
 
 type TargetNode implements Node {
     target: ServiceRef!
-    recordings: Recordings!
+    recordings: Recordings
     mbeanMetrics: MBeanMetrics!
```

Comment:
`mbeanMetrics` should also allow null responses for the same reasons as the mutations, I think.